test/optim/test_initializers.py

```diff
@@ -275,14 +275,14 @@ def test_gen_batch_initial_conditions_constraints(self):
             bounds = torch.tensor([[0, 0], [1, 1]], device=self.device, dtype=dtype)
             inequality_constraints = [
                 (
-                    torch.tensor([1], device=self.device, dtype=dtype),
+                    torch.tensor([1], device=self.device, dtype=torch.int64),
```

Comment:
```suggestion
                    torch.tensor([1], device=self.device, dtype=dtype),
```
Let me know if I'm missing something, but tests still pass for me with the old version so I'd rather leave it how it is